src/pages/lessons/[slug].tsx

```diff
@@ -209,7 +214,7 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
         'loaded',
         'viewing',
         'completed',
-        'showingNext',
+        // 'showingNext',
```

Comment:
was need to exclude `showingNext` state from `playerVisible` states to be able to get needed styles for overlay states.